__tests__/app.test.js

```diff
@@ -252,4 +252,63 @@ describe("GET all reviews", () => {
                 expect(reviews[0].review_id).toEqual(7);
             });
     });
+});
+
+describe("GET comments by review ID", () => {
+    describe("happy path :)", () => {
+        const testComment = {
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: expect.any(Number),
+        };
+
+        test("Ensure that the request comes back with 200 status and the appropriate body when the review ID is valid and there are comments to return", () => {
+            return request(app)
+                .get("/api/reviews/2/comments")
+                .expect(200)
+                .then(({ body }) => {
+                    const { comments } = body;
+                    expect(comments).toHaveLength(3);
+                    comments.forEach((comment) => {
+                        expect(comment).toEqual(
+                            expect.objectContaining(testComment)
```

Comment:
The only issue i have with this is that you are expecting ANY review_id, so we cannot ensure we are getting the comments for a specific article